packages/kbn-interpreter/src/server/get_plugin_paths.js

```diff
@@ -31,10 +31,17 @@ const canvasPluginDirectoryName = 'canvas_plugin';
 const isDirectory = path =>
   lstat(path)
     .then(stat => stat.isDirectory())
-    .catch(() => false);
+    .catch(() => false); // if lstat fails, it doesn't exist and is not a directory
 
 const isDirname = (p, name) => path.basename(p) === name;
 
+const filterDirectories = (paths, { exclude = false } = {}) => {
```

Comment:
if i understand this function correctly, it will return only directories if exclude is false and return only files if exclude = true ?

we could simplify it into:
```
return Promise.all(paths.filter(p => exclude ? !isDirectory(p) : isDirectory(p)));
```